cli/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "cli"
```

Comment:
`cli` does match the naming of our other binaries.  I suspect we want customers to use this as well, I'm thinking we'll then want to make this more descriptive like `mongosql-cli`.